src/Sidebar.js

```diff
@@ -72,7 +63,13 @@ function Sidebar() {
         <hr className="break-line" ></hr>
         <SidebarOptions Icon={AddIcon} title="Add Channels" />
         {channels.map((channel) => {
-          <SidebarOptions title={channel.name} id={channel.id} />
+          return (
+            <SidebarOptions
+              key={channel.id}
```

Comment:
This was bug 4. Items on a collection when rendering with react always must have the `key` prop.

Reply:
understood :-)